### PR TITLE
added bot heading to Shuffleboard

### PR DESCRIPTION
**Pull request by tharunS123**

### CODE REVIEW

File: src/main/java/team/gif/robot/Robot.java [hunk 1]
```diff
@@ -65,6 +67,7 @@ public void robotInit() {
         elevator = new Elevator();
         collector = new Collector();
         collectorPneumatics = new CollectorPneumatics();
+        pigeon = new Pigeon();
```
Comment (billknopfjr) [inline src/main/java/team/gif/robot/Robot.java]: We should change the pigeon class to include the CAN ID as an argument. I think it is hardcoded in the pigeon class to use RobotMap.

File: src/main/java/team/gif/robot/UI.java [hunk 2]
```diff
@@ -3,6 +3,7 @@
 import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
 import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardTab;
 
+import static com.ctre.phoenix.GadgeteerUartClient.GadgeteerProxyType.Pigeon;
```
Comment (billknopfjr) [inline src/main/java/team/gif/robot/UI.java]: Is this the correct import or should it be:
import team.gif.robot.subsystems.drivers.Pigeon;

File: src/main/java/team/gif/robot/UI.java [hunk 3]
```diff
@@ -12,5 +13,11 @@ public UI() {
         ShuffleboardTab shuffleboardTab = Shuffleboard.getTab("FRC2023");
 
         shuffleboardTab.add("Arm Ticks", arm.getTicks()); // displaying the ticks for the arm.
+
+        // Pigeon
+        shuffleboardTab.add("Bot Heading", (x) -> {
```
Comment (billknopfjr) [inline src/main/java/team/gif/robot/UI.java:17]: For now this is fine but we don't want to use a new tab in the final build

**Review by billknopfjr — CHANGES_REQUESTED** [review]

File: src/main/java/team/gif/robot/UI.java (hunk 2 above)
Comment (tharunS123) [reply src/main/java/team/gif/robot/UI.java]: import team.gif.robot.subsystems.drivers.Pigeon; This is right. Sorry.

File: src/main/java/team/gif/robot/UI.java (hunk 3 above)
Comment (tharunS123) [reply src/main/java/team/gif/robot/UI.java:17]: ok

**Pull request closed without merging**